azurerm/resource_arm_function_app.go

```diff
@@ -134,6 +144,16 @@ func resourceArmFunctionApp() *schema.Resource {
 							Optional: true,
 							Default:  false,
 						},
+						"use_32_bit_worker_process": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Default:  false,
```

Comment:
Based on [this comment](https://github.com/terraform-providers/terraform-provider-azurerm/issues/745#issuecomment-360933129) I believe this needs to be defaulted to `true` for Function Apps?